client/app/services/query.js

```diff
@@ -253,6 +253,12 @@ export class Parameter {
       }
     }
 
+    if (this.type === 'query' && !isEmptyValue(value)) {
+      if (this.multiValuesOptions && !isArray(value) && value !== null) {
```

Comment:
Don't think you need to check `value !== null` cause it's checked the line before.

Reply:
Yeah, I was a bit paranoid on this one :sweat_smile: